Drivers/Src/Display.c

```diff
@@ -23,8 +23,12 @@
 // Operational commands have no attribute and no operator, just a command and >= 0 arguments
 #define isOpCmd(cmd) (cmd.op == NULL && cmd.attr == NULL)
 
+#define RESTART_THRESHOLD 3 // number of times to restart before fault
```

Comment:
We should not need this here, since it is already defined in UpdateDisplay and unused here.